src/uk/gov/hmcts/contino/Kubectl.groovy

```diff
@@ -95,6 +101,10 @@ class Kubectl {
     this.steps.sh(script: "env AZURE_CONFIG_DIR=/opt/jenkins/.azure-${this.subscription} az aks get-credentials --resource-group ${this.resourceGroup} --name ${this.clusterName} --subscription  ${aksSubscription} -a ", returnStdout: true)
   }
 
+  def loginJenkins() {
+    this.steps.sh(script: "env AZURE_CONFIG_DIR=/opt/jenkins/.azure-jenkins az aks get-credentials --resource-group ${this.businessArea}-ptl-00-rg --name ${this.businessArea}-ptl-00-aks --subscription  ${this.jenkinsSubscription} -a ", returnStdout: true)
```

Comment:
don't guess, just set it as an environment variable like the others
https://github.com/hmcts/sds-flux-config/blob/master/apps/jenkins/jenkins/ptl/jenkins.yaml#L49-L52